src/smart-components/role/add-role-new/schema.js

```diff
@@ -158,7 +158,7 @@ export default (container) => ({
               component: 'cost-resources',
               name: 'cost-resources',
               // eslint-disable-next-line quotes
-              description: "Specify where you'd like to apply each cost permission selected int the previous step, suig the dropdown below.",
+              description: "Specify where you'd like to apply each cost permission selected int the previous step, using the dropdown below.",
```

Comment:
@PanSpagetka Any idea why the description isn't showing?

Reply:
I am fixing it here: https://github.com/RedHatInsights/insights-rbac-ui/pull/535